From inaky@linux.intel.com  Sun Jan 25 14:33:44 2009
From: Inaky Perez-Gonzalez <inaky@linux.intel.com>
Date: Thu,  8 Jan 2009 12:52:19 -0800
Subject: USB: add kernel-doc for wusb_dev in struct usb_device
To: linux-usb@vger.kernel.org
Cc: David Vrabel <david.vrabel@csr.com>, Randy Dunlap <randy.dunlap@oracle.com>, Greg Kroah-Hartman <gregkh@suse.de>
Message-ID: <1231447939-8158-1-git-send-email-inaky@linux.intel.com>


Reported by Randy Dunlap from a warning on the v2.6.29 merge window.

Signed-off-by: Inaky Perez-Gonzalez <inaky@linux.intel.com>
Cc: David Vrabel <david.vrabel@csr.com>
Cc: Randy Dunlap <randy.dunlap@oracle.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 include/linux/usb.h |    2 ++
 1 file changed, 2 insertions(+)

--- a/include/linux/usb.h
+++ b/include/linux/usb.h
@@ -418,6 +418,8 @@ struct usb_tt;
  * @autosuspend_disabled: autosuspend disabled by the user
  * @autoresume_disabled: autoresume disabled by the user
  * @skip_sys_resume: skip the next system resume
+ * @wusb_dev: if this is a Wireless USB device, link to the WUSB
+ *	specific data for the device.
  *
  * Notes:
  * Usbcore drivers should not set usbdev->state directly.  Instead use
